packages/grid/_modules_/grid/components/GridRow.tsx

```diff
@@ -17,12 +17,31 @@ export interface GridRowProps {
   children: React.ReactNode;
 }
 
+const useUtilityClasses = (ownerState) => {
```

Comment:
Can we type the `ownerState` ?

Reply:
All `ownerState` are properly typed now.